CHANGES.rst

```diff
@@ -20,6 +20,7 @@ Unreleased
     :class:`~validators.ValidationError` or
     :class:`~validators.StopValidation` to make a validation fail.
     :issue:`445`
+-   Filters can be inline. :issue:`128` :pr:`592`
```

Comment:
This needs to describe the change more, since it requires any existing `process` methods to accept `**kwargs`. Something like "`form.process()` will be passed an `extra_filters` parameter."